lightning/src/ln/channelmanager.rs

```diff
@@ -4926,6 +4942,11 @@ where
 				num_unfunded_channels += 1;
 			}
 		}
+		for (_, chan) in peer.inbound_v1_channel_by_id.iter() {
```

Comment:
I probably really need to define `Channel` better now. I had it as `FundedChannel` before, but because of the nomenclature we have here it didn't make sense.

So an `InboundV1Channel` becomes a `Channel` once the acceptor sends a `funding_signed` and hence starts out as not usable. The idea is that once we have a `ChannelMonitor` for a channel, then it becomes a `Channel`. 

Maybe `MonitoredChannel` is a better name?

Reply:
Eh, not sure if we want both `MonitoredChannel` and `ChannelMonitor`. Would it make sense to have an intermediary `FundedChannel` before becoming a `ConfirmedChannel`, or something similar?

Reply:
I think just documenting it will be fine for now, but those are good suggestions if we decide to split them. I feel like read/write could be tricky. Would have to put that in this PR too.